src/modm/platform/comp/stm32/module.lb

```diff
@@ -48,7 +68,22 @@ def prepare(module, options):
     if not device.has_driver("comp:stm32*"):
         return False
 
-    if not device.get_driver("comp")["type"] in ["stm32-v1.3", "stm32-tsmc90_cube"]:
+    """
+    Existing comparator IPs:
+        "stm32-tsmc90_cube"
+        "stm32-tsmc90_dory_cube"
+        "stm32-tsmc90_g4_rockfish_cube"
+        "stm32-tsmc90_h7_cube"
+        "stm32-tsmc90_orca128_cube"
+        "stm32-tsmc90_orca512_cube"
+        "stm32-tsmc90_orcazero_cube"
+        "stm32-v1.0"
+        "stm32-v1.2"
+        "stm32-v1.3"
+        "stm32-v3.4"
+        "stm32-v3.6"
+    """
+    if not device.get_driver("comp")["type"] in ["stm32-v1.3", "stm32-tsmc90_cube", "stm32-tsmc90_g4_rockfish_cube"]:
```

Comment:
Hm.

Reply:
Better idea?

Reply:
Rename it in modm-devices, but I just made it incompatible with the latest identifier extension, so hm. Guess we'll fix it later.